packages/validator/src/validator.test.js

```diff
@@ -0,0 +1,884 @@
+        "security": {
+          "mySecurityScheme": [],
+          "mySecurityScheme2": []
+        }
+      }`;
+      expect(validator.isApiSpecsValid(validArraySizesAPISpec)).toMatchObject({ valid: true, messages: [] });
+
+      const exceededArraySizeAPISpec = `{
+        "servers": [
+            {
+                "url":  "https://myapi.com/api"
+            },
+            {
+                "url":  "https://myapi.com/api"
+            }
+        ],
+        "paths": {
+            "/myPath": {
+              "get": {
+                "parameters": [
+                  {
+                    "name": "myParameter",
+                    "in": "path"
+                  },
+                  {
+                    "name": "myParameter",
+                    "in": "query"
+                  },
+                  {
+                    "name": "myParameter",
+                    "in": "header"
+                  }
+                ]
+              }
+            },
+            "/myPath2": {
+              "get": {
+                "parameters": [
+                  {
+                    "name": "myParameter",
+                    "in": "path"
+                  },
+                  {
+                    "name": "myParameter",
+                    "in": "query"
+                  },
+                  {
+                    "name": "myParameter",
+                    "in": "header"
+                  }
+                ]
+              }
+            }   
+        },
+        "components": {
+            "securitySchemes": {
+              "mySecurityScheme": {
+                "type": "apiKey",
+                "name": "X-MY-API-KEY",
+                "in": "query"
+              },
+              "mySecurityScheme2": {
+                "type": "apiKey",
+                "name": "X-MY-API-KEY",
+                "in": "header"
+              }
+            }
+        },
+        "security": {
+              "mySecurityScheme": [],
+              "mySecurityScheme2": []
+          }
+      }`;
+      expect(validator.isApiSpecsValid(exceededArraySizeAPISpec)).toMatchObject({
+        valid: false,
+        messages: [sizeExceededMessage('servers', 1)],
+      });
+    });
+
+    test('JSON validity test', () => {
+      const smallValidAPISpec = `{
+        "servers": [],
+        "paths": {},
+        "components": {
+            "securitySchemes": {}
+        },
+        "security": {}
+      }`;
+      expect(validator.isApiSpecsValid(smallValidAPISpec)).toMatchObject({ valid: true, messages: [] });
+
+      const invalidJSONAPISpec = `{
+        "servers": [],
+        "paths": {},
+        "components": {
+            "securitySchemes": {}
+        },
+        "security": {}`;
+      expect(validator.isApiSpecsValid(invalidJSONAPISpec)).toMatchObject({
+        valid: false,
+        messages: [{ level: 'error', message: 'SyntaxError: Unexpected end of JSON input' }],
+      });
+    });
+
+    test('missing parameters', () => {
```

Comment:
Just for consistency, I think let's use `it()` instead of `test()` for test cases - they do the exact same thing. All of the other packages in the monorepo use `it()`

Reply:
Replaced in e678ec5